single.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+get_header();
+
+// If a featured image is assigned to the post, display as a background image.
+if ( spine_has_background_image() ) {
+	$background_image_src = spine_get_background_image_src();
+
+	?><style> html { background-image: url('<?php echo esc_url( $background_image_src ); ?>'); }</style><?php
+}
+?>
+
+<main>
+
+<?php
+
+get_template_part('parts/headers-news');
```

Comment:
@charlesjdillon Every blog post or page across the IP ecosystem that is not using the builder template will now load this template. Is that desired or is this for a more specific use case?

It may be desirable to find a way to conditionally output the NEWS H1 inside `parts/headers.php`.